core/src/avm1/globals/bevel_filter.rs

```diff
@@ -0,0 +1,676 @@
+//! flash.filter.BevelFilter object
+
+use crate::avm1::activation::Activation;
+use crate::avm1::error::Error;
+use crate::avm1::function::{Executable, FunctionObject};
+use crate::avm1::object::bevel_filter::{BevelFilterObject, BevelFilterType};
+use crate::avm1::{AvmString, Object, TObject, Value};
+use enumset::EnumSet;
+use gc_arena::MutationContext;
+
+pub fn constructor<'gc>(
+    activation: &mut Activation<'_, 'gc, '_>,
+    this: Object<'gc>,
+    args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error<'gc>> {
+    let distance = args
+        .get(0)
+        .unwrap_or(&4.0.into())
+        .coerce_to_f64(activation)?;
+
+    let angle = args
+        .get(1)
+        .unwrap_or(&44.9999999772279.into())
```

Comment:
I wonder if the almost-45 default means the angle is actually stored as radians (in an f32?), and the getters/setters convert to/from degrees? Not sure, something to keep in mind and maybe test more thoroughly someday.